docker-compose.yml

```diff
@@ -50,6 +50,7 @@ services:
     labels:
       - traefik.http.routers.dissolution-api.rule=Host(`chs-dev`) && PathPrefix(`/dissolution-api/`)
     environment:
+      - APPEALS_API_MONGODB_URL=mongodb://mongo/dissolutions
```

Comment:
APPEALS_API? Shouldn't this be DISSOLUTION_API?

Reply:
We are sharing the rebel1 environment with appeals, and their env variable is named this so we can reuse it.